react/components/Prefetch/LRUCache.ts

```diff
@@ -0,0 +1,124 @@
+class Node<T> {
```

Comment:
I hate to be that person... but since we're adding a custom LRU lib, we should at least create some tests for it.

Reply:
tests added